site/en/blog/building-for-the-web/index.md

```diff
@@ -0,0 +1,447 @@
+systems make this a lot more attractive for platform-specific apps, albeit at a 15–30% commission.
+For apps built using
+[Trusted Web Activities](/docs/android/trusted-web-activity/) and
+delivered through the Google Play Store, developers can now use the
+[Payment Request API](https://developer.mozilla.org/docs/Web/API/Payment_Request_API) and the
+new
+[Digital Goods API](/docs/android/trusted-web-activity/receive-payments-play-billing/)
+to integrate with Google Play Billing.
+
+<details>
+  <summary>How big a challenge is this?</summary>
+  <p>
+    When you profit from the convenience of app store billing or in-app purchases, at the same time you also leave a part of your benefits on the table as a commission. As a matter of fact, some apps that <em>are</em> published to app stores even ask their users to make the purchase off-store.
+  </p>
+</details>
+
+#### Hiring or retraining developers
+
+From personal experience through talking to many of Google's partners, a lot of
+[companies struggle with hiring great web developers](https://medium.com/javascript-scene/why-hiring-is-so-hard-in-tech-c462c3230017).
+The talent shortage is real, and recruiting costs are high, which is why startups commonly hire
+in-house recruiters who often approach recruiting with a shotgun approach that has not has not helped
+the reputation of recruiters with IT professionals. Also, companies often already employ teams of Android
+and/or iOS developers that they cannot just retrain to become web developers. Creating a PWA
+requires a high level of specialization that not all web developers can offer.
+
+<details>
+  <summary>How big a challenge is this?</summary>
+  <p>
+    In the current economic situation, hiring <em>any</em> kind of developer is difficult. Hiring someone with web development skills is, in comparison to other web technology skills, still easier, which (in part) also explains the popularity of app frameworks such as React Native, Ionic, Flutter, and others that are based on web technologies.
+  </p>
+</details>
+
+#### Existing apps legacy (and migrating the user base)
+
+It is not unusual for companies to have made massive investments in platform-specific apps, and
+giving up these investments, as well as a user base acquired over time (not to speak of the vanity
+install statistics), is not easy. Apparently, starting from scratch, even when a company has an
+existing website, appears very unattractive in comparison, but sometimes it does
+[happen](https://www.glossy.co/ecommerce/gone-fishin-patagonia-bids-farewell-to-mobile-app/).
+
+<details>
+  <summary>How big a challenge is this?</summary>
+  <p>
+    Vanity is vanity, but once you have set up new, potentially more meaningful, tracking metrics than number of app installs such as <a href="https://web.dev/betty-crocker/#results">increase of indicators of purchase intent</a>, you can start tracking those instead.
+  </p>
+</details>
+
+#### Compatibility with relevant browsers
```

Comment:
@tomayac @jpmedley I'm very concerned that this section could be interpreted as misrepresenting the Interop project, which is not covering Capabilities APIs at all. Given this article is linked to a piece about Fugu, and Interop is not looking at Fugu APIs (it's mostly CSS), it could be interpreted that there is an expectation the Interop 2022 will help to get Fugu APIs into other browsers, that's not the case.

Reply:
Fair point. I'll remove this.

Reply:
Removed.